src/main/java/ch/njol/skript/effects/EffBreakNaturally.java

```diff
@@ -18,67 +18,100 @@
  */
 package ch.njol.skript.effects;
 
-import org.bukkit.block.Block;
-import org.bukkit.event.Event;
-import org.bukkit.inventory.ItemStack;
-import org.eclipse.jdt.annotation.Nullable;
-
 import ch.njol.skript.Skript;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Break Block")
-@Description({"Breaks the block and spawns items as if a player had mined it",
-		"\nYou can add a tool, which will spawn items based on how that tool would break the block ",
-		"(ie: When using a hand to break stone, it drops nothing, whereas with a pickaxe it drops cobblestone)"})
-@Examples({"on right click:", "\tbreak clicked block naturally",
-		"loop blocks in radius 10 around player:", "\tbreak loop-block using player's tool",
-		"loop blocks in radius 10 around player:", "\tbreak loop-block naturally using diamond pickaxe"})
-@Since("2.4")
+@Description({
+	"Breaks the block and spawns items as if a player had mined it",
+	"\nYou can add a tool, which will spawn items based on how that tool would break the block ",
+	"(ie: When using a hand to break stone, it drops nothing, whereas with a pickaxe it drops cobblestone)",
+	"\nWhen 'with effect' is used, the sound and particles while breaking the block is produced.",
+	"\nWhen 'and drop experience' is used, XP will drop at the block if it usually does so."
+})
+@Examples({
+	"on right click:",
+		"\tbreak clicked block naturally",
+	"loop blocks in radius 10 around player:",
+		"\tbreak loop-block using player's tool",
+	"loop blocks in radius 10 around player:",
+		"\tbreak loop-block with effects naturally using diamond pickaxe and drop xps"
+})
+@Since("2.4, INSERT VERSION (effects, drop xps)")
+@RequiredPlugins("Paper 1.15+ (effects), Paper 1.17+ (effects without tool), Paper 1.19+ (drop xps)")
 public class EffBreakNaturally extends Effect {
+
+	private static final boolean hasMethod115 = Skript.methodExists(Block.class, "breakNaturally", CollectionUtils.array(ItemStack.class, boolean.class), boolean.class);
+	private static final boolean hasMethod117 = Skript.methodExists(Block.class, "breakNaturally", boolean.class, boolean.class);
+	private static final boolean hasMethod119 = Skript.methodExists(Block.class, "breakNaturally", CollectionUtils.array(ItemStack.class, boolean.class, boolean.class), boolean.class);
 	
```

Comment:
Unpopular opinion but maybe don't add `with effects` for 1.15, hear me out, if you require defining an item for this it basically means there's no way both can be optional as `with effects` would require `using %-itemtype%`

Reply:
If you wanted to keep this then there would need to be 2 patterns
```
break %blocks% with effect[s] [naturally] using %-itemtype%
break %blocks% [naturally] [using %-itemtype%]
```
but even then this is really ugly

Reply:
If you need the effects part to be conditional on the item part, you could put them into the same optional.
e.g. `[a [b]]` could be `a` or `a b` but never just `b`.

Reply:
With my suggestion below about rearranging the `with effects` part to be after `using tool` this would be possible indeed 👌

Reply:
Merged Moderocky's and Ayham's suggestion.